docs/proposals/006-scheduler/README.md

```diff
@@ -0,0 +1,150 @@
+# Gateway API Scheduler
+
+Authors: @kfswain, @smarterclayton
+
+## Proposal Status
+ ***Draft***
+
+## Table of Contents
+
+<!-- toc -->
+
+-   [Summary](#summary)
+-   [Goals](#goals)
+-   [Non-Goals](#non-goals)
+-   [Proposal](#proposal)
+    -   [Personas](#personas)
+    -   [Requirements](#requirements)
+    -   [Design](#design)
+    -   [Alternatives](#alternatives)
+- [FAQ](#faq)
+- [Open Questions](#open-questions)
+    
+<!-- /toc -->
+
+## Summary
+
+This proposal defines the inference gateway scheduler subsystem and constrains its scope. The scheduler is responsible for determining which endpoints the load balancer should route requests to. Future proposals may extend its scope.
+
+## Goals
+
+- The scheduler should be reasonably fast - decide request mapping to endpoints within O(10ms) on average
+- The scheduler should be effective - requiring little configuration out of the box to get great performance
+- The scheduler should be maintainable - new in-tree features should compose cleanly
+- The scheduler should be forkable - downstream consumers should expect some stability of interface
+- The scheduler should be educatable - extending the [model server protocol](../003-model-server-protocol/) with new metrics or adding a new source of data should be minimally invasive
+- The scheduler should be replaceable - the reference endpoint picker implementation should support delegating scheduling decisions per pool to an alternative **replacement scheduler**
+
+## Non-Goals
+
+- Dynamic reconfiguration of the reference scheduler algorithms at runtime
+- Being a general scheduler framework for load balancing
```

Comment:
I ca n understand this is tageted to be a generic inference scheduling framework instead of common web service